src/components/App.js

```diff
@@ -1,11 +1,89 @@
 import React from 'react';
+import Temp from './Temp.js'
+// import CarouselPicture from './CarouselPicture.js';
```

Comment:
Try to keep the commented-out code to a minimum, if it's needed, then no worries!